### PR TITLE
Fix CORS for st2auth and st2api

### PR DESCRIPTION
**Pull request by enykeev**

### CODE REVIEW

**Comment by Kami** [comment]
LGTM, +1

File: st2auth/st2auth/config.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def _register_app_opts():
     api_opts = [
         cfg.ListOpt('allow_origin', default=['http://localhost:3000'],
             help='List of origins allowed'),
+        cfg.BoolOpt('serve_webui_files', default=True,
```
Comment (Kami) [inline st2auth/st2auth/config.py]: Would if we should actually move it to st2common.config since we already have some common API options there...

Comment (enykeev) [inline st2auth/st2auth/config.py]: Lets do it in another PR. It can wait.
